src/vcpkg/binarycaching.cpp

```diff
@@ -620,10 +620,10 @@ namespace
                     print2(res2.output);
                 }
 
-                return res2.exit_code;
+                return std::move(res2);
```

Comment:
Without move I get 
```
[ 17%] Building CXX object CMakeFiles/vcpkglib.dir/src/vcpkg/binaryparagraph.cpp.o
/Users/runner/work/1/s/src/vcpkg/binarycaching.cpp:623:24: error: local variable 'res2' will be copied despite being returned by name [-Werror,-Wreturn-std-move]
                return res2;
                       ^~~~
/Users/runner/work/1/s/src/vcpkg/binarycaching.cpp:623:24: note: call 'std::move' explicitly to avoid copying
                return res2;
                       ^~~~
                       std::move(res2)
/Users/runner/work/1/s/src/vcpkg/binarycaching.cpp:626:20: error: local variable 'res' will be copied despite being returned by name [-Werror,-Wreturn-std-move]
            return res;
                   ^~~
/Users/runner/work/1/s/src/vcpkg/binarycaching.cpp:626:20: note: call 'std::move' explicitly to avoid copying
            return res;
                   ^~~
                   std::move(res)
2 errors generated.

```

Reply:
Huh. OK!